script.js

```diff
@@ -0,0 +1,67 @@
+function invokeRZP()
+{
+    var data = razorpay_script_vars.data;
+
+    var setDisabled = function(id, state)
+    {
+        if (typeof state === 'undefined')
+        {
+            state = true;
+        }
+
+        var elem = document.getElementById(id);
+
+        if (state === false)
```

Comment:
@riagarg5 please run prettier against this file. We're using that to standardize all our javascript codebases. https://github.com/prettier/prettier